src/api/uninstall.ts

```diff
@@ -100,6 +101,15 @@ export async function uninstallInContext (
     bin: opts.bin,
   })
 
+  if (opts.shamefullyFlatten) {
+    const deps = R.keys(currentShrinkwrap.specifiers).map(
+      pkgName => `${pkgName}@${currentShrinkwrap.specifiers[pkgName]}`
+    )
+    if (deps.length > 0) {
+      await installPkgs(deps, Object.assign({}, opts, {lock: false, internalSkipLock: true}))
```

Comment:
why is it needed? Isn't `currentShrinkwrap.specifiers ` what was just installed?